### ISSUE TITLE
Calling someone at the same time they call you results in neither client being able to accept either call request.

### ISSUE THREAD

**Issue by AverytheFurry** [vector-im/element-web#20259]
### Steps to reproduce

1. Call a friend at the same time they call you
2. Observe  the "connecting..." dialogue
3. The call never starts unless you hang up and recall.

#### What did you expect?
It to either show the incoming call (could use the existing dialogue for when you're called from another individual while in a call) or automatically answer (since I sent a call request at the same time, to the exact same person)
#### What happened instead?
The call waits indefinitely. I've had situations happen where we both hit call at the same time, hit hang up, hit call at the same time, etc.

### Operating system

Windows

### Application version

Element version: 1.9.7 Olm version: 3.2.8

### How did you install the app?

https://community.chocolatey.org/packages/element-desktop

### Homeserver

yiff.social

### Will you send logs?

No

**Comment by SimonBrandner** [comment]
Hmm, glare resolution works fine for me, could you please share your logs?

**Comment by AverytheFurry** [comment]
> Hmm, glare resolution works fine for me, could you please share your logs?

I don't feel comfortable sharing the data on everyone I talk to & stuff, which is included in logs.

**Comment by chronos1970** [comment]
I can confirm, this bug occurs for me on a regular basis even when both parties are using the windows client.

**Comment by SimonBrandner** [comment]
I am having quite some trouble reproducing this, I am suspicious of decryption doing something weird, so I am wondering if you tend to exchange messages with the other person before actually calling them?

**Comment by AverytheFurry** [comment]
> I am having quite some trouble reproducing this, I am suspicious of decryption doing something weird, so I am wondering if you tend to exchange messages with the other person before actually calling them?

Yes, how would we know to call without messaging eachother lol?

**Comment by SimonBrandner** [comment]
> > I am having quite some trouble reproducing this, I am suspicious of decryption doing something weird, so I am wondering if you tend to exchange messages with the other person before actually calling them?
> 
> Yes, how would we know to call without messaging eachother lol?

I just wanted to make sure :P

**Comment by SimonBrandner** [comment]
Do you have an idea of how often this happens? I think I managed to repro in 1 out of 10 maybe and have no idea what is the difference in that case. Is it also 1 in 10 or something similarly low for you?

**Comment by toger5** [comment]
I can reproduce this very consistently (andorid with web at least) (web with web seems to work fine.)
I need to start the call on android before it shows me, that there is a call incoming. Than it does not connect

**Comment by SimonBrandner** [comment]
> I can reproduce this very consistently (andorid with web at least) (web with web seems to work fine.) I need to start the call on android before it shows me, that there is a call incoming. Than it does not connect

The problems between Android <-> Web/Desktop are Android's fault, the glare resolution there seems to be lacking :/

**Comment by toger5** [comment]
I did manage to break this on web as well, but as you mentioned already not reproducible...

**Comment by chronos1970** [comment]
I would like to point out that for me this seems to happen when the call suddenly drops, and both parties try to recall right after. The most common cause for calls dropping for me has been when both parties actively share and stop sharing their screens.

**Comment by SimonBrandner** [comment]
> The most common cause for calls dropping for me has been when both parties actively share and stop sharing their screens.

Hmm, I've been unable to repro this as well :thinking:

**Comment by AverytheFurry** [comment]
Here's an example of this happening, note "connecting" on the earlier call
![image](https://user-images.githubusercontent.com/47131867/148690237-6f6e1317-f664-4205-8f87-b1274fd60d5b.png)

**Comment by Doomsdayrs** [comment]
This occurs as well for me frequently

**Comment by Doomsdayrs** [comment]
The best way to handle this is a check to see if which request was first, then to simply drop ones one request and send an accept response

**Comment by w4tsn** [comment]
This is happening for me as well on Element Desktop trying to connect to other Element Desktop clients. My company is using the vector.im Offering.

```
Element version: 1.9.9
Olm version: 3.2.8
```

**Comment by gabmert** [comment]
@SimonBrandner 
Just happened to me again: 

Manjaro Desktop (Element: 1.10.8, Olm: 3.2.8)
to 
Windows Desktop

both tried to call at the same time, but could only get through after both stopped their call and only one of us called.

I'm happy to provide logs, is there a way to not post them to the whole internet? Else I'll have to get a bit creative to anonymise them. Or is there a tool to do that?

**Comment by turt2live** [comment]
I don't think this was ever fixed. https://github.com/vector-im/element-web/issues/2199 appears to have been closed incorrectly.

**Comment by DavyLandman** [comment]
True, we get this quite often. Between 2 desktop clients.
